src/main/java/com/minecolonies/apiimp/initializer/ModModelTypeInitializer.java

```diff
@@ -0,0 +1,124 @@
+package com.minecolonies.apiimp.initializer;
+
+import com.minecolonies.api.client.render.modeltype.BipedModelType;
+import com.minecolonies.api.client.render.modeltype.CitizenModel;
+import com.minecolonies.api.client.render.modeltype.ModModelTypes;
+import com.minecolonies.api.client.render.modeltype.registry.IModelTypeRegistry;
+import com.minecolonies.coremod.client.model.*;
+
+public class ModModelTypeInitializer
```

Comment:
wrong method formatting in file

Reply:
Do you mean you want everything instantiated first, then registered?

Reply:
No
{

}